docs/SYNC_MODE.md

```diff
@@ -34,14 +34,14 @@ Taps should allow users to choose which streams and fields to replicate. The fol
 
 | Metadata Keyword | Description  |
 | ----------------- | ------- |
-| `inclusion` | Only applies to fields.  If this is set to `automatic`, the field should be replicated.  If this is set to `unsupported`, the field should not be replicated.  Can be written by a tap during discovery |
+| `inclusion` | Only applies to fields.  If this is set to `automatic`, the field should be replicated. If this is set to `available`, the field can be either selected or left unselected.  If this is set to `unsupported`, the field should not be replicated.  Can be written by a tap during discovery |
 | `selected` | If this is set to `True`, the stream (empty breadcrumb), or field should be replicated.  If `False`, the stream or field should be omitted.  This metadata is written by services outside the tap. |
-| `available` | The field may or may not be `selected`. |
 
-Streams shouldn’t be selected by default. We want them to have an `inclusion: available` property. Each field needs to have its own metadata object that labels the field as `inclusion: automatic` or `inclusion: available`.
+Streams shouldn’t be selected by default. They must have an `inclusion: available` property. Each field needs to have its own metadata object that labels the field as `inclusion: automatic` or `inclusion: available`.
```

Comment:
The first time I read this, I didn't see a difference and got confused. Maybe it should break into two paragraphs here, like this (some small edits too)

> Streams shouldn’t be selected by default. They must have an `inclusion: available` property.
>
> Additionally, each field needs to have its own metadata object that labels the field as `inclusion: automatic`, `inclusion: available`, or `inclusion: unsupported`.